src/components/Button.js

```diff
@@ -1,12 +1,23 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './Button.css';
 
-function Button() {
+function Button({ buttonName }) {
   return (
     <div className="Button" id="button">
-      <p> Calculator App: Hello world!</p>
+      <p>
+        { buttonName}
+      </p>
     </div>
```

Comment:
- [ ] We can use the `button` element here instead of a `div` element since this component actually represents a button. This way we can also remove the `p` element since the button element accepts a text value.